### ISSUE TITLE
Order in Z axis is not saved

### ISSUE THREAD

**Issue by Arri0** [logisim-evolution/logisim-evolution#501]
Hello, 
when I save Circuit Appearance, order of shapes in "Z axis" is not saved. Also when font is bold, after loading it is not.

Original design:
![LogisimBug1](https://user-images.githubusercontent.com/17547286/107865501-76474080-6e67-11eb-8c73-390227f9e9ce.png)

Design after loading circuit from file:
![LogisimBug2](https://user-images.githubusercontent.com/17547286/107865508-819a6c00-6e67-11eb-99b8-00e111b1a353.png)

A am using prebuilt binary (logisim-evolution_3.4.1-1_amd64.deb).

**Comment by Emil-DV** [comment]
I noticed this as well and it is quite a headache since I'm going for a sunken 'battle ship' panel look.
I was able to set up the build environment and found that the issue was with a "sort" function that was added somewhere along the way for easier testing via .circ file compares (so say the comments in the code).  I commented out the call to sort (logisim-evolution-develop\src\main\java\com\cburch\logisim\file\XmlWriter.java line 197) and the Z order works now.
